packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -352,6 +393,10 @@ class _RenderVectorGraphics extends RenderBox {
     // This is safe to do because the widget itself is wrapped in a repaint
     // boundary.
     context.canvas.drawPicture(_pictureInfo.picture);
+
+    if (color != null) {
+      context.canvas.drawColor(color!, blendMode);
```

Comment:
This doesn't have bounds so i don't think it works. I think the right approach is with either drawPaint or save layer.

Reply:
(I mean it shows up correctly, but I think it applies to the entire canvas? Which might actually be OK since we clip it